test/Makefile.in

```diff
@@ -64,20 +66,28 @@ test_harness:       $(TESTHARNESS_OBJS)
 	@echo "- linking $@"
 	$(Q)$(CC) -L../src $(LDFLAGS) $(UNWINDLIB) -o $@ $(TESTHARNESS_OBJS) $(LIBS) -lmtev
 
-busted:
-	@busted --version >/dev/null 2>/dev/null || \
-		(echo "Please instead busted: luarocks install busted" && false)
+mtevbusted-script:	mtevbusted-script.in
```

Comment:
Mainly so that one could override it.. but we could do that in autoconf as well.